### PR TITLE
feat: Create and display profile name, username, bio, and picture components

### PR DESCRIPTION
**Pull request by dipasukhaedbert**
- Add the name, username, bio, and profile picture components
- Enable user to live-edit the name and bio, without connecting to firebase yet
- Display name and username according to the user's input during the sign-up phase

### CODE REVIEW

File: src/components/name-output/name-output.component.jsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React, { useRef, useState } from "react";
+import EditName from "../name/name.component";
+import "./name-output.styles.css";
+
+const firebase = require("firebase");
```
Comment (felixhalim) [inline src/components/name-output/name-output.component.jsx:5]: This is not needed. We have imported this from index.firebase.js

File: src/components/name-output/name-output.component.jsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+const { db, auth } = require("../../firebase/index.firebase");
+
+const NameOutput = () => {
+  const inputRef = useRef();
+  const [content, setContent] = useState("");
+
+  var user = firebase.auth().currentUser;
```
Comment (felixhalim) [inline src/components/name-output/name-output.component.jsx:12]: Why not use auth.currentUser?

File: src/components/name/name.component.jsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from "react";
+import "./name.styles.css";
+
+const firebase = require("firebase");
```
Comment (felixhalim) [inline src/components/name/name.component.jsx:4]: Not needed

File: src/components/username-output/username-output.component.jsx [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import React from "react";
+import "./username-output.styles.css";
+
+const firebase = require("firebase");
```
Comment (felixhalim) [inline src/components/username-output/username-output.component.jsx:4]: Why don't you import the auth method from index.firebase?

File: src/components/username-output/username-output.component.jsx [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+
+const UsernameOutput = () => {
+  var user = firebase.auth().currentUser;
```
Comment (felixhalim) [inline src/components/username-output/username-output.component.jsx:7]: use auth method from index.firebase.js

File: src/components/name-output/name-output.component.jsx (hunk 1 above)
Comment (dipasukhaedbert) [reply src/components/name-output/name-output.component.jsx:5]: Well noted.

**Comment by felixhalim** [comment]
Please fix

**Pull request closed without merging**